modules/2_qc_filtering/rna/qc_filtering.qmd

```diff
@@ -254,253 +195,167 @@ if ("ensembl_chr" %in% colnames(feature_metadata)) {
 
-if (length(orig_idents) <= 10) {
-  cat("::: {layout-ncol=2}", sep="\n")
-} else {
-  cat("::: {layout-ncol=1}", sep="\n")
-}
+<details class="details_box">
+  <summary class="details_box">Which cells can be considered to be of low quality?</summary>
 
-for (i in seq(plist)) {
-  chunk_filled =  knitr::knit_expand(text=chunk_template, type=barcode_qc[i], i=i, caption=captions[i])
-  if(interactive()) {
-    EvalKnitrChunk(chunk_filled)
-  } else {
-    chunk_filled = knitr::knit_child(text=chunk_filled, envir=environment(), quiet=TRUE)
-    cat(chunk_filled, sep='\n')
-  }
-}
+On the one hand, low-quality cells such as dying, degraded and damaged cells, and empty droplets will often have very few genes ("nFeature_RNA") and low total counts ("nCount_RNA"). On the other hand, cell doublets may show an aberrantly high number of genes. Since the total number of reads detected within a cell typically strongly correlates with the number of unique genes, we can focus on the number of unique genes for filtering. In addition, damaged cells often exhibit high mitochondrial ("percent_mt") or spike-in ("percent_ercc") content. As mitochondria have their own membranes, their RNA is often the last to degrade in damaged cells and can thus be found in high numbers. However, it is important to keep in mind that different cell types and cells isolated from various species may differ in their number of expressed genes and metabolism. For example, stem cells may express a higher number of unique genes, and metabolically active cells may express a higher number of mitochondrial genes. 
+</details>
 
-cat(":::", sep="\n")
-```
+<details class="details_box">
+  <summary class="details_box">Impact of low-quality cells on downstream analyses</summary>
 
+First of all, low-quality cells of different cell types might cluster together due to similarities in their damage-induced gene expression profiles. This leads to artificial intermediate states or trajectories between subpopulations that would otherwise be distinct. Second, low-quality cells might mask relevant gene expression changes. The main differences captured by the first principal components will likely be based on cell quality rather than the underlying biology, reducing the power of dimensionality reduction. Likewise, highly variable genes might just be genes that differ between low- and high-quality cells. Lastly and equally important, the low number of total counts in low-quality cells might lead to artificial upregulation of genes due to wrong scaling effects during the normalisation process. 
+</details>
 
 ```{r}
-#| label: preprocess_rna_qc_cor_plots
```

Comment:
If we decide to use the wording 'barcode' synonymous for cells, nuclei and spots, can we please add comments to various places in the code to remind me and other developers what the word means?